Display/server.py

```diff
@@ -0,0 +1,77 @@
+import sys
+import time
+import json
+import random
+
+import zmq
+
+from patienttubingdescriptorcalculator \
+    import PatientTubingDescriptorCalculator
```

Comment:
you don't _have_ to comply to the 80 character line limit now that we have widescreen monitors with more than 1024x768 resolution.

Reply:
Ah, but PEP8 says _79_ columns

Reply:
and I'm saying that line can be ignored because we have wider screens than 1024x768.